snippets/post_install_network_config

```diff
@@ -77,6 +77,9 @@ mv /etc/sysconfig/network.cobbler /etc/sysconfig/network
 # Also set the hostname now, some applications require it
 # (e.g.: if we're connecting to Puppet before a reboot).
 /bin/hostname $hostname
+        #if $osversion == "rhel7"
```

Comment:
I don't reproduce when I set the hostname in cobbler System config and deploy with kickstart on tftp (so I have a network --bootproto=dhcp --device=eth0 --hostname myhostname) 
That tested with both RHEL/CentOS 7.3. and using the sample_end.ks

Basically if I set the hostname in kickstart, I have the related /etc/hostname updated, no matter if it is set on DNS or not.